torch_geometric/nn/conv/gcn_conv.py

```diff
@@ -171,8 +177,10 @@ def __init__(
         improved: bool = False,
         cached: bool = False,
         add_self_loops: bool = True,
+        t: float = 1.0,
         normalize: bool = True,
-        bias: bool = True,
+        bias: bool = False,
+        weight_initializer: str = "normal",
```

Comment:
```suggestion
        weight_initializer: str = "glorot",
```
Keep it as `glorot` for compatibility.

Reply:
In this case, would it be possible to add a note that "normal" is an option to the documentation? Maybe with a pointer to our paper, highlighting the importance of this init?